maps/bagelstation.dmm

```diff
@@ -12737,7 +12737,10 @@
 "eKW" = (/obj/item/clothing/under/doomguy,/turf/simulated/floor/airless,/area)
 "eKX" = (/obj/structure/grille/broken,/turf/simulated/floor/plating/airless,/area)
 "eKY" = (/obj/structure/grille,/turf/simulated/floor/airless{broken = 1; icon_state = "floorscorched2"},/area)
-
+"eKZ" = (/obj/effect/mine/kick,/mob/living/simple_animal/hostile/faithless/cult,/turf/simulated/floor/airless{tag = "icon-cult"; icon_state = "cult"},/area/mine/unexplored)

```

Comment:
I started noticing stuff like this after all those vendors mysteriously appeared in an unrelated mapping PR